tdrs-backend/tdpservice/users/management/commands/generate_test_users.py

```diff
@@ -0,0 +1,72 @@
+"""generate_test_users command."""
```

Comment:
This is only for generating users for the testing suite and won't be used in production